### PR TITLE
Precaching

### PR DESCRIPTION
**Pull request by lordjabez**
This PR enables the precaching of values without calling the underlying function.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -148,6 +148,24 @@ See here for an example:
...
+
+If you want to load a value into the cache without calling the underlying function, this can be done with the `precache_value` function.
+
+.. code-block:: python
+
+  @cachier()
+  def add(arg1, arg2):
+    return arg1 + arg2
+
+  foo.precache_value(2, 2, value_to_cache=5)
```
Comment (shaypal5) [inline README.rst:162]: This should be `add.precache_value`...

**Review by shaypal5 — CHANGES_REQUESTED** [review]
Except for a tiny fix in the documentation of the feature, awesome PR!

**Comment by lordjabez** [comment]
Fixed!

**Review by shaypal5 — APPROVED** [review]

**Comment by shaypal5** [comment]
Released in [v2.1.0](https://github.com/python-cachier/cachier/releases/tag/v2.1.0)

**Pull request merged**